src/javascript/generator.mts

```diff
@@ -48,10 +49,41 @@ function generator(
     return name;
   }
 
+  const includes = (config.includes || []).map(
+    (pattern) => new RegExp(pattern),
+  );
+  const excludes = (config.excludes || []).map(
+    (pattern) => new RegExp(pattern),
+  );
+
+  function getModifiedPathValue(endPoint: string, value: PathItem) {
+    return Object.entries(value).filter(
+      ([method, options]: [string, SwaggerRequest]) => {
+        const { operationId } = options;
+
+        const serviceName = generateServiceName(
+          endPoint,
+          method,
+          operationId,
+          config,
+        );
+
+        const matchesInclude =
+          !includes.length || includes.some((regex) => regex.test(serviceName));
+
+        const matchesExclude = excludes.some((regex) =>
+          regex.test(serviceName),
+        );
+
+        return matchesInclude && !matchesExclude;
+      },
+    );
+  }
+
   try {
     Object.entries(input.paths).forEach(([endPoint, value]) => {
       const parametersExtended = value.parameters as Parameter[] | undefined;
-      Object.entries(value).forEach(
+      getModifiedPathValue(endPoint, value).forEach(
```

Comment:
`getModifiedPathValue ` doesn't make sense and could be `getFilteredMethods`

Reply:
getModifiedPathValue changed to getFilteredMethods